book/website/communication/citable/citable-cite.md

```diff
@@ -67,10 +68,14 @@ To make your code citable, you can use the integration between [Zenodo](https://
 - On GitHub, click the *Create a new release* button.
 Zenodo should automatically be notified and should make a snapshot copy of the current state of your repository (just one branch, without any history), and should also assign a persistent identifier (DOI) to that snapshot.
 - Use the DOI in any citations of your software and tell any collaborators and users to do the same!
+
```

Comment:
Is this intended?

Reply:
This will be updated. Currently in the test mode.

Reply:
I assume not -- I suspect it's an edit slipped into this PR.

Reply:
```suggestion
```